open-craft/ansible-playbooks
# [BB-3847] Update prometheus role to support LimitNOFILE

## Description

This PR updates the `ernestas-poskus.prometheus` role from the submodule repository https://github.com/ernestas-poskus/ansible-prometheus.

It fixes `too many open files error` in prometheus.net.opencraft.hosting. A [PR](https://github.com/ernestas-poskus/ansible-prometheus/pull/105) was created on the third-party repo to solve the issue, which got merged. This PR is to update the corresponding submodule to include the fix.

## Supporting information

https://tasks.opencraft.com/browse/BB-3847

## Testing instructions

1. Pull this PR
2. Deploy a prometheus instance
3. The `prometheus.service` file should contain `LimitNOFILE` set to `infinity`

## Deadline

"None" 

## Other information

**Reviewers**
